deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml

```diff
@@ -31,7 +31,7 @@ resources:
     zone: SET_THE_ZONE
     # "1.X": picks the highest valid patch+gke.N patch in the 1.X version
     # https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.zones.clusters
-    cluster-version: "1.10"
+    cluster-version: "1.11"
     # Set this to v1beta1 to use beta features such as private clusterss
     # and the Kubernetes stackdriver agents.
     gkeApiVersion: v1
```

Comment:
Do you want to move this one to cluster.jinja, and make a mapping from GKE version?
1.10 -> v1
1.11 -> v1beta1

Reply:
These should not be mapped 1:1. And I think we need 1.11 anyway to support multiple CRD versions.

Reply:
Or make:
```gkeApiVersion: v1beta1```
to have test coverage